05_3D_Analyst/3D_Analyst.Rmd

```diff
@@ -0,0 +1,5 @@
+# 3D Analyst Toolbox
```

Comment:
I would suggest naming the file describing the toolbox "index.Rmd" (e.g. 05_3D_Analyst/index.Rmd)

Reply:
Careful! this does not work, since all files are moved to the top level directory and there can be only one "index.html". So let's leave the name "3D_Analyst.Rmd" etc.